crates/superposition_types/migrations/2024-12-17-100613_workspaces/up.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TYPE public.workspace_status AS ENUM (
```

Comment:
Could this have gone in a separate PR with the CRU for this table?

Reply:
Sure I can split it